### ISSUE TITLE
Idea for improved accessibility with asset pages

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1983]
This is pretty directly related to #1870 as well as similar situations where we have links that do searches with icons that take one to the external name authority.  I asked a question to the webaim list, and got an off-list response:

> I think the problem (for all repositories) is that the metadata is often handling two tasks: describing the items and providing ways to discover similar or related items. I'm not sure there is a way to get around that, so I would be inclined to leave the links for the metadata as the text that is viewed on the page currently (the subject terms). It might be helpful to provide some text that explains where the links go somewhere on the page (maybe in an expander under the "Details" column?), but I don't think increasing the verbosity of the links themselves would ultimately help screen reader users.
> 
> However, for the University of Oregon link, it might be helpful to provide more context for users. If possible, I would be inclined to provide text that becomes visible on mouse hover or focus and is appended to the link for screen reader and other assistive technology users. So something like `<a href="/catalog?f%5Binstitution_label_sim%5D%5B%5D=University+of+Oregon&amp;locale=en">University of Oregon <span class="mostlyhidden">search all items from this institution</span></a>`, where the `<span>` text is served up on hover and focus and read to screen reader users after the visible text. One key would be to include any supplemental text after the visible text, that's a WCAG 2.1 requirement and is particularly useful for users browsing with voice activation software, like Dragon speech recognition. I think something similar for the little info "i" links would also make sense.

To summarize:

- For asset pages, where there are a huge number of links, consider adding instructions somewhere that explains the links' purpose, since it's not evident. The links themselves should be kept as-is to avoid unnecessary verbosity.
- For a collection landing page, where there aren't so many links, consider text that shows up on hover/focus that explains the link's purpose.
  - The text should always be part of the link so screen readers see it, and always *after* the visible text for speech recognition where a user might say "click university of oregon"
- The various "i" links could have hover/focus text explaining their purpose - which is basically what they need to have for screen reader users anyway

**Comment by azle** [comment]
Update 12/30/2024 see related error: https://github.com/OregonDigital/OD2/issues/1870
